jenkins/Jenkinsfile

```diff
@@ -185,32 +155,63 @@ pipeline {
-          SECRET_TOKEN_LOCAL = "${env.SECRET_TOKEN_BASE}${NASA_MATURITY}"
-
-          build job: "${env.DOWNSTREAM_JOB}", parameters: [
-              [$class: 'StringParameterValue', name: 'MATURITY', value: "${MATURITY_LOCAL}"],
-              [$class: 'StringParameterValue', name: 'DEPLOY_NAME', value: "${DEPLOY_NAME_LOCAL}"],
-              [$class: 'StringParameterValue', name: 'DAAC_REPO', value: "${params.DAAC_REPO}"],
-              [$class: 'StringParameterValue', name: 'DAAC_REF', value: "${params.DAAC_REF}"],
-              [$class: 'StringParameterValue', name: 'AWS_REGION', value: "${params.AWS_REGION}"],
-              [$class: 'StringParameterValue', name: 'AWS_CREDS', value: "${AWS_CREDS_LOCAL}"],
-              [$class: 'StringParameterValue', name: 'CMR_CREDS_ID', value: "${CMR_CREDS_LOCAL}"],
-              [$class: 'StringParameterValue', name: 'URS_CREDS_ID', value: "${URS_CREDS_LOCAL}"],
-              [$class: 'StringParameterValue', name: 'SECRET_TOKEN_ID', value: "${SECRET_TOKEN_LOCAL}"],
-              [$class: 'StringParameterValue', name: 'MAKE_COMMAND', value: "${params.MAKE_COMMAND}"],
-              [$class: 'StringParameterValue', name: 'DB_ADMIN_PASSWORD', value: "${env.DB_ADMIN_PASSWORD}"],
-          ]
-        }
+        deploy(
+          DEPLOY_NAME: params.DEPLOY_NAME,
+          AWS_REGION: params.AWS_REGION,
+          MATURITY: 'prod',
+          CONTAINER_TAG: params.CORE_REF,
+          MAKE_TARGET: params.MAKE_TARGET,
+          AWS_CREDS: "${env.AWS_CREDS_BASE}prod",
+          CMR_CREDS: "${env.CMR_CREDS_BASE}PROD",
+          URS_CREDS: "${env.URS_CREDS_BASE}PROD",
+          DB_CREDS: "${env.DB_CREDS_BASE}prod",
+          SECRET_TOKEN_ID: "${env.SECRET_TOKEN_BASE}prod"
+        )
       }
     }
+  } // stages
+} // pipeline
+
+
+def deploy(params) {
+  withEnv([
+    "DEPLOY_NAME=${params.DEPLOY_NAME}",
+    "AWS_REGION=${params.AWS_REGION}",
+    "MATURITY=${params.MATURITY}",
+    "CONTAINER_TAG=${params.CONTAINER_TAG}",
+    "MAKE_TARGET=${params.MAKE_TARGET}"
+  ]) {
+    withCredentials([
+      [$class: 'AmazonWebServicesCredentialsBinding', credentialsId: params.AWS_CREDS],
+      usernamePassword(credentialsId: params.CMR_CREDS, passwordVariable: 'CMR_CREDS_PSW', usernameVariable: 'CMR_CREDS_USR'),
+      usernamePassword(credentialsId: params.URS_CREDS, passwordVariable: 'URS_CREDS_PSW', usernameVariable: 'URS_CREDS_USR'),
+      usernamePassword(credentialsId: params.DB_CREDS, passwordVariable: 'DB_CREDS_PSW', usernameVariable: ''),
+      string(credentialsId: params.SECRET_TOKEN_ID, variable: 'TOKEN_SECRET')
+    ])  {
+      sh '''export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+            export AWS_ACCOUNT_ID_LAST4=$(echo -n $AWS_ACCOUNT_ID | tail -c 4)
+            cd $WORKSPACE/CIRRUS-core
+            docker run --rm   --user `id -u` \
+                              --env DAAC_DIR="/CIRRUS-ASF" \
```

Comment:
This should be `/CIRRUS-DAAC` instead of `CIRRUS-ASF`

Reply:
Good catch